src/router/index.ts

```diff
@@ -113,6 +116,11 @@ const router = new VueRouter({
 
 router.beforeEach((to, from, next) => {
   if (to.matched.some(record => record.meta.requiresAuth)) {
+    if ((to.name === PageNames.BridgeTransactionsHistory || to.name === PageNames.BridgeTransaction) && isWalletConnected() && (web3Util.getEthUserAddress() === '' || web3Util.getEthUserAddress() === 'undefined')) {
```

Comment:
A lot of checks. 1st, you don't need `web3Util.getEthUserAddress() === '' || web3Util.getEthUserAddress() === 'undefined'` as I mentioned above.
2nd, please replace `to.name === PageNames.BridgeTransactionsHistory || to.name === PageNames.BridgeTransaction` with
`[PageNames.BridgeTransactionsHistory, PageNames.BridgeTransaction].includes(to.name as PageNames)`

Reply:
and also you can create an array above something in common place (maybe in consts) like `ChildBridgePages` and use it here